ols/src/llms/llm_loader.py

```diff
@@ -78,25 +81,24 @@ def __init__(
 
         # return empty dictionary if not defined
         self.llm_params = params if params else {}
-        self.llm = None
-        self._set_llm_instance()
+        self.llm: LLM = self._llm_instance()
 
-    def _set_llm_instance(self):
+    def _llm_instance(self):
```

Comment:
@tisnik shouldn't this specify the return type since we seem to be moving to doing that everywhere?

Reply:
yes it would be better do done in explicitly there, correct! (Mypy can infer the type, but return type is part of function/method documentation)

Reply:
you want to submit a follow up for that?

for us python newbies there's gonna be a lot of copying+pasting from the existing code examples, so we should try to ensure the existing code reflects how we want new code to look

Reply:
https://github.com/openshift/lightspeed-service/pull/194

Reply:
ah, looks like you already did in https://github.com/openshift/lightspeed-service/pull/194